package/wpe/wpe.mk

```diff
@@ -147,9 +147,15 @@ ifeq ($(BR2_PACKAGE_WPE_ENABLE_JS_MEMORY_TRACKING),y)
 	WPE_FLAGS += -DENABLE_JS_MEMORY_TRACKING=ON
 endif
 
-WPE_CONF_OPT = -DPORT=WPE -G Ninja \
-	-DCMAKE_BUILD_TYPE=$(BUILDTYPE) \
-	$(WPE_FLAGS)
+ifeq ($(BR2_PACKAGE_WPE_GENERATE_ECLIPSE_PROJECT),y)
+	WPE_CONF_OPT = -DPORT=WPE -G 'Eclipse CDT4 - Ninja' \
```

Comment:
Instead of changing the whole CONF_OPT variable perhaps use a new NINJA_GENERATOR variable?